delegates.txt

```diff
@@ -281,6 +281,7 @@ Patrick Soquet (PST)
 Paul Leather (PLR)
 Paul Leathers (PL)
 Pedram Emrouznejad (PED)
+Peter Frivalszky-Mayer (FMP)
```

Comment:
Would you prefer `PFM` to match the others?

Reply:
Sure, of course. Thanks!